### PR TITLE
Bugfix - Updated Lorem content at /financial-report

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated placeholder text at `/financial-report` with true text from content team.
 
## Changes
 
- intro at /financial-report
 
## Testing
 
Navigate to:

- /financial-report

## Review
 
- @jameshupp 
- @juidai 
- @sebworks 
- @anselmbradford 
 
## Preview
 
screen shot 2015-03-20 at 3 40 00 pm

## Notes
 
None

### CODE REVIEW

File: budget/financial-report/index.html [hunk 1]
```diff
@@ -15,7 +15,10 @@
     <div class="block block__flush-top block__sub">
         <h1>Financial Report & Update</h1>
         <p class="h3">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.
+            Each fiscal year, the Chief Financial Officer releases a report on the Bureau’s finances.
+            This report analyzes how we allocate funds and how’ve done at meeting our stated goals for
```
Comment (anselmbradford) [inline budget/financial-report/index.html]: `and how’ve done` should be `and how we’ve done`, yes?

Comment (jimmynotjim) [inline budget/financial-report/index.html]: I would assume so, but I didn't write it. @jameshupp?

Comment (jameshupp) [inline budget/financial-report/index.html]: Yeah, sorry. Not sure what happened there.

Comment (jimmynotjim) [inline budget/financial-report/index.html]: Thanks @jameshupp. Updated and ready again for review.

**Comment by anselmbradford** [comment]
:+1:

**Pull request merged**